### PR TITLE
Modified SubmissionForm.razor

### PR DESCRIPTION
**Pull request by giaou**
Due to security problem, InputFile component doesnt allow users to retrieve path of selected files. So instead of read the path, Im reading the name of the file.

### CODE REVIEW

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 1]
```diff
@@ -125,4 +144,47 @@
         public string? InputFile { get; set; }
     }
 
+    // private ElementReference inputFileRef;
+    // string filePath = "C:/Users/lesli/OneDrive/Desktop/text/file.txt";
+    // string filePath1 = "C:/Users/lesli/OneDrive/Desktop/text/file1.txt";
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:149]: please remove this

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 2]
```diff
@@ -125,4 +144,47 @@
...
+    {
+
+        JobModel.InputFile = e.File.Name;
+        /*var buffer = new byte[e.File.Size];
+
+        await e.File.OpenReadStream().ReadAsync(buffer);
+        var fileType = e.File.ContentType;
+        var fileData = $"data:{fileType};base64,{Convert.ToBase64String(buffer)}";
+        JobModel.InputFile = fileData;*/
+        //await File.AppendAllTextAsync(filePath, "\n\n\n" + e.File.Name);
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:161]: please remove this

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 3]
```diff
@@ -125,4 +144,47 @@
...
+        System.Diagnostics.Debug.WriteLine(JobModel.InputFile);
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:162]: we would not want to ship this to prod, please remove

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 4]
```diff
@@ -125,4 +144,47 @@
...
+    }
+
+    public void OnInputSupplementChange(InputFileChangeEventArgs e)
+    {
+        JobModel.InputSupplementDir = e.File.Name; 
+        System.Diagnostics.Debug.WriteLine(JobModel.InputSupplementDir);
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:168]: please remove

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 5]
```diff
@@ -125,4 +144,47 @@
...
+
+    }
+
+
+    public void OnRuntimeConfigChange(InputFileChangeEventArgs e)
+    {
+         JobModel.RuntimeConfigFile = e.File.Name;
+        System.Diagnostics.Debug.WriteLine(JobModel.RuntimeConfigFile);
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:176]: please remove

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 6]
```diff
@@ -125,4 +144,47 @@
...
+
+    }
+
+
+    public void OnOutputFileChange(InputFileChangeEventArgs e)
+    {
+        JobModel.OutputDir = e.File.Name;
+        System.Diagnostics.Debug.WriteLine(JobModel.OutputDir);
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:184]: please remove

File: src/Presentation/Components/UI/SubmissionForm.razor [hunk 7]
```diff
@@ -125,4 +144,47 @@
...
     }
 
+    // private ElementReference inputFileRef;
+    // string filePath = "C:/Users/lesli/OneDrive/Desktop/text/file.txt";
+    // string filePath1 = "C:/Users/lesli/OneDrive/Desktop/text/file1.txt";
+
+    public void OnInputFileChange(InputFileChangeEventArgs e)
+    {
+
+        JobModel.InputFile = e.File.Name;
```
Comment (ducmvo) [inline src/Presentation/Components/UI/SubmissionForm.razor:154]: this is good for now to just save the file name.
We will implement Egnyte file picker to fix this

**Review by ducmvo — CHANGES_REQUESTED** [review]
Can we have some unit tests?

File: src/Presentation/Components/UI/SubmissionForm.razor (hunk 1 above)
Comment (giaou) [reply src/Presentation/Components/UI/SubmissionForm.razor:149]: sure will!! i will remove all the commented out stuff

Comment (giaou) [reply src/Presentation/Components/UI/SubmissionForm.razor:149]: removed

File: src/Presentation/Components/UI/SubmissionForm.razor (hunk 2 above)
Comment (giaou) [reply src/Presentation/Components/UI/SubmissionForm.razor:161]: removed

File: src/Presentation/Components/UI/SubmissionForm.razor (hunk 3 above)
Comment (giaou) [reply src/Presentation/Components/UI/SubmissionForm.razor:162]: removed

**Pull request merged**